### PR TITLE
Include URLs in Java code templates for easier customization

### PR DESCRIPTION
**Pull request by JaroslavTulach**
While working on #2960 I forgot to modify Java code templates. That's annoying as these templates are exactly something that needs customization:

image

Clicking the URL in editor opens the template.

### CODE REVIEW

**Review by junichi11 — APPROVED** [review]
Great!

**Comment by sdedic** [comment]
Could it be possible to define a standard variable, like ${open_url} for the template expansion ?

File: java/java.hints/test/unit/src/org/netbeans/modules/java/hints/suggestions/ImplementMethodsTest.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public void testSimple1() throws Exception {
                 .assertOutput("package test;\n" +
                               "public abstract class Test implements Runnable {\n" +
                               "    public void run() {\n" +
-                              "        throw new UnsupportedOperationException(\"Not supported yet.\"); //To change body of generated methods, choose Tools | Templates.\n" +
+                              "        throw new UnsupportedOperationException(\"Not supported yet.\"); // Generated from nbfs://nbhost/SystemFileSystem/Templates/Classes/Code/GeneratedMethodBody\n" +
```
Comment (JaroslavTulach) [inline java/java.hints/test/unit/src/org/netbeans/modules/java/hints/suggestions/ImplementMethodsTest.java:47]: There doesn't seem to be any way to treat the comment as `.*` without specifying the exact content, right @jlahoda?

**Comment by JaroslavTulach** [comment]
> Could it be possible to define a standard variable, like ${open_url} for the template expansion ?

There is `${.data_model["org.openide.filesystems.FileObject"].getPath()` in https://github.com/apache/netbeans/pull/2960/files#diff-498697d41fe5fc687f18b0e779cb675567c29ef83e224dac048a58b43a9a9ceeR3 - I am not sure if that is better, however.

File: java/java.hints/test/unit/src/org/netbeans/modules/java/hints/suggestions/ImplementMethodsTest.java (hunk 1 above)
Comment (JaroslavTulach) [reply java/java.hints/test/unit/src/org/netbeans/modules/java/hints/suggestions/ImplementMethodsTest.java:47]: Let's assume there is no better way and all tests had to be modified in ba90ee1

**Pull request merged**